src/EPR.RegulatorService.Facade.API/Controllers/InviteRegulatorController.cs

```diff
@@ -87,7 +91,8 @@ public async Task<IActionResult> CreateInviteEnrollment([FromBody] RegulatorInvi
             }
             catch (Exception e)
             {
-                _logger.LogError($"Error when creating the invite regulator user for id {request.UserId}");
```

Comment:
The standard way to use Microsoft logger in other project is as follows
_logger.LogError("Error when creating the invite regulator user for id {UserId}", request.UserId);